woodwork/indexers.py

```diff
@@ -42,7 +42,6 @@ def _process_selection(selection, original_data):
         elif _is_dataframe(original_data):
             # Selecting a single column from a DataFrame
             schema = original_data.ww.schema.columns[selection.name]
-            schema.semantic_tags = schema.semantic_tags - {'index'} - {'time_index'}
             if schema.use_standard_tags:
                 schema.semantic_tags |= schema.logical_type.standard_tags
```

Comment:
Now that we're keeping the `index` tag, I'm not sure that we want to add standard tags back like this (and at the table accessor getitem) since at the Woodwork table level, you either have standard tags or the index tag but not both. 

Not sure if maybe at the column level it does make sense to have both. @thehomebrewnerd thoughts?

Reply:
Yeah, my thought going into this was that when we grab a column from the dataframe, the column schema for the individual column should match the schema that is in the dataframe for that column - that way no mater where the user looks they see the same info.

tldr: I vote for not adding back in standard tags.

Reply:
✅   ceccc4d